django/cantusdb_project/main_app/signals.py

```diff
@@ -193,11 +121,139 @@ def generate_volpiano_intervals(volpiano_notes):
     )
 
 
-def update_prefix_field(instance):
+def generate_volpiano_notes(volpiano) -> None:
```

Comment:
Some discrepancies between type annotations, docstring, and return statement...

Reply:
Thank you!